src/checkpatch_wrapper.sh

```diff
@@ -76,12 +76,12 @@ function execute_checkpatch()
       continue
     fi
 
-    cd "$kernel_root"
+    cd "$kernel_root" || exit
```

Comment:
Add a message that says why it fails, and also exit with a proper exit code.

Reply:
Running `exit` this way utilizes the last commands exit code, therefore therefore propagating the error code from the failed `cd` command. The `exit` command doesn't take any arguments apart from a custom exit code, shellcheck suggests the following option for displaying custom messages:
```cd foo || { echo "Failure"; exit 1; }```
I could change to it if that's preferred.

Reply:
@rodrigosiqueira, dont know if you missed this last reply from rubens